wordpress-mobile/WordPress-iOS
# Zendesk Mobile: add support for Jetpack and self-hosted users.

Fixes #9170 

**NOTES:**
- Your local `.wpcom_app_credentials` must be up-to-date as it contains Zendesk credentials. If you have not, refer to the Field Guide for instructions.
- Accessing 'Help' from Signup or Login is not working properly yet.
- There is a distinct lack of error handling. That's on my to-do list for later.

**To test:**
- Be sure you are _not_ logged in with a WP account.
- Connect multiple self-hosted sites. 
- Select any site.
  - Create a Support ticket.
  - Verify that ticket appears in the 'My Tickets' list.
- Select another site.
  - Verify that ticket still appears in the 'My Tickets' list.
  - Create another Support ticket.
  - Verify both tickets appears in the 'My Tickets' list.

In short, no matter which self-hosted site is selected, 'My Tickets' should show all tickets.

@jleandroperez - would you have time to review this? Thank you!
